lib/ProductOpener/Food.pm

```diff
@@ -175,12 +174,9 @@ Output modifier reference.
 
 =cut
 
-sub normalize_nutriment_value_and_modifier($$) {
+sub normalize_nutriment_value_and_modifier($value_ref, $modifier_ref) {
 
-	my $value_ref = shift;
-	my $modifier_ref = shift;
-
-	$$modifier_ref = undef;
+	$modifier_ref = undef;
```

Comment:
```suggestion
	${$modifier_ref} = undef;
```
I believe the tests are failing because the reference instead of the value is set to undef here.

Reply:
Oh, thanks for fixing it! I thought the $$ was a typo :sweat_smile: